licences-specs/src/test/groovy/uk/gov/justice/digital/hmpps/licences/specs/assessment/StandardConditionsVersionsSpec.groovy

```diff
@@ -0,0 +1,68 @@
+package uk.gov.justice.digital.hmpps.licences.specs.assessment
+
+import geb.spock.GebReportingSpec
+import spock.lang.Shared
+import spock.lang.Stepwise
+import uk.gov.justice.digital.hmpps.licences.pages.TaskListPage
+import uk.gov.justice.digital.hmpps.licences.pages.assessment.LicenceConditionsAdditionalPage
+import uk.gov.justice.digital.hmpps.licences.pages.assessment.LicenceConditionsStandardPage
+import uk.gov.justice.digital.hmpps.licences.pages.assessment.LicenceConditionsSummaryPage
+import uk.gov.justice.digital.hmpps.licences.util.Actions
+import uk.gov.justice.digital.hmpps.licences.util.TestData
+
+@Stepwise
+class StandardConditionsVersionsSpec extends GebReportingSpec {
```

Comment:
Happy for my Geb testing to be critqued, never done it before so I'm sure there'll be some mistakes here 😂